model_compression_toolkit/core/common/network_editors/actions.py

```diff
@@ -333,3 +336,37 @@ def apply(self, node: BaseNode, graph, fw_info):
                     raise Exception('Unknown quantization method for weights')
 
                 qc.weights_quantization_cfg.set_weights_quantization_fn(weights_quantization_fn)
+
+
+class ReplaceLayer(BaseAction):
```

Comment:
High-level question (I'm just not sure what is the expected behavior): 
When using this action to change a node's type--what should happen to the node's quantization configuration?
For example, if the new type is a type of node that we do not quantize, or if the new type has a different quantization configuration defined in the TPC, what is the expected behavior? because the current implementation does not pay attention to this part (updating the node's quantization configuration).